mmv1/products/compute/RegionNetworkEndpointGroup.yaml

```diff
@@ -243,6 +246,8 @@ properties:
       - cloud_run
       - app_engine
       - serverless_deployment
+    allow_empty_object: true
+    send_empty_value: true
```

Comment:
```suggestion
```
Same as `cloud_run`, we require at least one of `function` and `url_mask` specified